docs/understanding-tracking-design/managing-your-data-structures/ui/json-editor/index.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Managing data structures with the JSON Editor"
+sidebar_label: "Using the JSON Editor"
+sidebar_position: 1
+---
+For more complex data structures that require heavy nesting and more advanced data types. 
```

Comment:
```suggestion
:::info

The JSON editor is ideal for more complex data structures that require heavy nesting and more advanced data types. For simple data structures use ... [link].

:::
```

I would also put a variation of that in the page description (in the frontmatter above). This would show in the doc cards on the parent page, and also helps with SEO.